08-transport.Rmd

```diff
@@ -28,36 +30,41 @@ It is therefore unsurprising that transport researchers have long turned to geoc
 This chapter provides an introduction to geographic analysis of transport systems.
 We will explore how movement patterns can be understood at multiple geographic levels, including:
 
-- **Areal units**: transport can be understood simply in terms of zonal aggregates such as the main mode and average distance of trips made people living in a particular zone.
+<!-- What is the main mode? I don't know the term. -->
```

Comment:
What is the main mode? I don't know the term. Reading on I understood what you meant but maybe a little clarification here would be helpful.

Reply:
Good suggestion, will do.

Reply:
👍